src/game_interpreter_battle.cpp

```diff
@@ -605,12 +606,42 @@ bool Game_Interpreter_Battle::CommandManiacControlAtbGauge(lcf::rpg::EventComman
 	return true;
 }
 
-bool Game_Interpreter_Battle::CommandManiacChangeBattleCommandEx(lcf::rpg::EventCommand const&) {
+bool Game_Interpreter_Battle::CommandManiacChangeBattleCommandEx(lcf::rpg::EventCommand const& com) {
 	if (!Player::IsPatchManiac()) {
 		return true;
 	}
 
-	Output::Warning("Maniac Patch: Command ChangeBattleCommandEx not supported");
+	// 1 row removed
+	bool actorCommandFlags = com.parameters[0];
+
+	lcf::Data::battlecommands.easyrpg_disable_row_feature = actorCommandFlags;
+
+	// 10000 lose added
+	// 01000 win added
+	// 00100 escape removed
+	// 00010 auto removed
+	// 00001 fight removed
+	int partyCommandFlags = com.parameters[1];
+
+	lcf::Data::system.easyrpg_battle_options.clear();
+	for (size_t i = 0; i < Scene_Battle::BattleOptionType::Lose + 1; i++)
+	{
+		bool partyCommandFlag = partyCommandFlags & (1 << i);
+		bool flagIsSet = i > 2;
+
+		if (partyCommandFlag == flagIsSet)
+		{
+			lcf::Data::system.easyrpg_battle_options.push_back(i);
+		}
+	}
+
+	auto& scene = Scene::instance;
+	Scene_Battle* sceneBattle = dynamic_cast<Scene_Battle*>(scene.get());
```

Comment:
We have rtti disabled on our homebrew platforms (no ``dynamic_cast`` available).

The typical idiom we use is (has the advantage that it will even work when the battle scene is not top of the stack. Though don't think this can happen here):

```cpp
auto* scene_battle = static_cast<Scene_Battle*>(Scene::Find(Scene::Battle)).get();

// Rest stays the same
```

Reply:
Good to know. I've ended up changing my cast to static, but keeping the rest the same since we're guaranteed to be inside a battle with this function call. I tested it by calling this function from outside battle and it works as intended (by never getting called). 

I would have taken your suggestion but it resulted in a compiler error and I wasn't sure it was worth spending the time to figure it out:
![image](https://github.com/user-attachments/assets/6da33143-e6fd-4a7f-b79a-7861dbe08de8)

However, if you feel strongly about making the change, just let me know and I can figure out the exact syntax.

Reply:
Sorry messed up the braces. Correct is ``auto* scene_battle = static_cast<Scene_Battle*>(Scene::Find(Scene::Battle).get());``

But yeah this code is indeed never running outside of the battle as this is code inside the interpreter.